homeassistant/components/group/media_player.py

```diff
@@ -109,7 +109,7 @@ def __init__(self, unique_id: str | None, name: str, entities: list[str]) -> Non
         """Initialize a Media Group entity."""
         self._name = name
         self._state: str | None = None
-        self._supported_features: int = 0
+        self._attr_supported_features = 0
```

Comment:
Should this be set as a class instance instead?

Reply:
After checking, it's already the case in MediaPlayerEntity so removed the default here.